client/scss/layouts/_home.scss

```diff
@@ -1,31 +1,9 @@
-.homepage header {
-  @apply w-text-primary;
-
-  display: flex;
-  flex-flow: row nowrap;
-  gap: 1em;
-
-  .col1 {
-    width: 50px;
-    margin-inline-end: 1em;
-    padding: 0;
-
-    // make way for the nav-menu button on mobile
-    @include media-breakpoint-down(xs) {
-      position: relative;
-      inset-inline-start: $mobile-nav-indent;
+.homepage {
+  header {
+    .header__title {
+      inset-inline-start: -1.5em;
```

Comment:
To simplify things I opted for a shared header styles approach only, see other comment below.